### ISSUE TITLE
feat: redesign site with more modern interface

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keymanweb.com#119]

**Comment by Meng-Heng** [comment]
Meeting and Progress: https://docs.google.com/document/d/1kpocWkDos3Qrh3LVQvbWN9-L4nKA6HhRMPQJ4utw5us/edit?usp=sharing

**Comment by Meng-Heng** [comment]
Currently putting this on hold. 
Am I supposed to wait on keymanweb running locally errors to be resolved to get this to work accordingly?
Both are running:
![image](https://github.com/user-attachments/assets/ce0ea21e-7b0b-4717-b124-e3becf72394c)

The stylings still does not appear (BOOTSTRAP_VERSION=v0.4, the only version that got the page to load).
![image](https://github.com/user-attachments/assets/22089acd-c39a-4017-9b6b-0367c086657c)

I used to run keymanweb just fine during updating the logo PR: https://github.com/keymanapp/keymanweb.com/pull/120. Once, returned from working on the shared-site, other bootstrap_version I tried running are always showing:
![image](https://github.com/user-attachments/assets/d8ac95d8-4388-4227-967e-3c9e368f621c)

**Comment by ermshiperete** [comment]
See also #87, #84
